frontend/src/lib/lemon-ui/LemonTag/LemonTag.scss

```diff
@@ -1,48 +1,99 @@
 .LemonTag {
-    font-size: 0.75rem;
-    font-weight: var(--font-medium);
+    align-items: center;
     background: var(--border);
-    padding: 0.125rem 0.25rem;
     border-radius: var(--radius);
-    display: inline-flex;
-    align-items: center;
     color: var(--default);
+    display: inline-flex;
+    font-size: 0.75rem;
+    font-weight: var(--font-medium);
     line-height: 1rem;
+    padding: 0.125rem 0.25rem;
     white-space: nowrap;
 
+    .posthog-3000 & {
+        background: none;
+        border-radius: calc(var(--radius) * 0.75);
+        border-style: solid;
+        border-width: 1px;
+        font-size: 0.688rem;
```

Comment:
I've noticed varying font sizes in a couple of places around our app e.g. buttons. Do we need to narrow it down to a handful?